### ISSUE TITLE
Use Github Environments to gate Production Releases

### ISSUE THREAD

**Issue by jflo** [hyperledger/besu#6862]
To better restrict the deployment of docker containers and the use of the 'latest' tag, we could use GitHub Actions Environments to ensure that multiple parties approve a deployment.

https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment

**Comment by jframe** [comment]
@jflo Should this issue stay open? Do we still want to work on this.

**Comment by jflo** [comment]
Yes, we still have a lot of potential to improve the security around releases.
